### PR TITLE
Add Gelma's typos from "co" to "cy"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -14252,6 +14359,8 @@ contiuguous->contiguous
 contiuing->continuing
 contiuous->continuous
 contiuously->continuously
+contnet->content
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:14362]: Any typos of `continent` out there?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -15438,6 +15571,7 @@ countours->contours, counters,
 countr->counter, contour, country, county,
 countrie's->countries, countries', country's,
 countrs->counters, contours, countries,
+countrys->countries
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:15574]: Also `countys->counties`?

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -15599,9 +15735,11 @@ creaetive->creative
 creaetor->creator
 creaetors->creators
 creaets->creates
+creaing->creating, creasing, creaking, creaming,
 creasoat->creosote
 creastor->creator
 creatation->creation
+creatd->created
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:15742]:
```suggestion
creatd->created, create,
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -15841,6 +15986,9 @@ cuestioned->questioned
 cuestions->questions
 cuileoga->cuileog
 culiminating->culminating
+culmulative->cumulative
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:15989]: Add all verbals forms of `culmulate` and `culiminate`?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -11583,8 +11583,10 @@ collpsible->collapsible
 collpsing->collapsing
 collsion->collision
 collsions->collisions
+collum->column
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:11586]: These are all too rare.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -12479,6 +12502,7 @@ compitability->compatibility
 compitable->compatible
 compitent->competent
 compitible->compatible
+complaince->compliance, complaints,
```
[the remark this answers is not in the document]
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:12505]: I couldn't find this typo.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**